lib/config.ts

```diff
@@ -8,6 +8,8 @@ const baseConfig = {
   facilitatorStartBlock: 0,
 };
 
+export type NetworkName = 'ethereum-rinkeby' | 'ethereum-mainnet';
```

Comment:
I'd recommend using the `SupportedNetwork` type from here: https://github.com/with-backed/backed-interface/blob/main/lib/config.ts#L16